language/Portuguese_Brazil/strings.po

```diff
@@ -1672,28 +1672,30 @@ msgstr " CANETAS"
 #: src/frontend/mame/ui/auditmenu.cpp:96
 #, c-format
 msgid "Audit ROMs for %1$u machines marked unavailable?"
-msgstr ""
+msgstr "Auditoria de ROMs para %1$u máquinas marcada como indisponível?"
 
 #: src/frontend/mame/ui/auditmenu.cpp:99
 #, c-format
 msgid "Audit ROMs for all %1$u machines?"
-msgstr ""
+msgstr "Auditoria de ROMs para todas %1$u máquinas? "
 
 #: src/frontend/mame/ui/auditmenu.cpp:104
 #, c-format
 msgid "(results will be saved to %1$s)"
-msgstr ""
+msgstr "(resultados irão ser salvos para %1$s)"
 
 #: src/frontend/mame/ui/auditmenu.cpp:130
 #, c-format
 msgid ""
 "Auditing ROMs for machine %2$u of %3$u...\n"
 "%1$s"
 msgstr ""
+"Auditando ROMs para máquina %2$u of %3$u...\n"
```

Comment:
Does "machine 40 of 50" really translate to Portuguese as "máquina 40 of 50" or is that an oversight?  (I would've thought it would be "40 de 50", but I can't actually speak Portuguese.)